### PR TITLE
Feat: new design!

### PR DESCRIPTION
**Pull request by modelm**

### CODE REVIEW

File: frontend-graphql/src/components/Footer.js [hunk 1]
```diff
@@ -1,30 +1,12 @@
 import React from 'react';
+import { ReactComponent as Logo } from '../static/images/postlight-logo.svg';
```
Comment (alexiakl) [inline frontend-graphql/src/components/Footer.js]: I am getting an error here that is crashing the GraphQL frontend:
```
Module not found: Can't resolve '../static/images/postlight-logo.svg' in '/home/node/app/src/components'
```

File: frontend-graphql/src/components/Category.js [hunk 2]
```diff
@@ -72,25 +73,39 @@ class Category extends Component {
...
               </div>
             ))}
-            <div className="f6 lh-copy gray" />
           </div>
         </div>
       </div>
     );
   }
-}
+  }
```
Comment (alexiakl) [inline frontend-graphql/src/components/Category.js]: looks like indentation is off, same applies to the `export` line just below this line

**Comment by CurtisBelt** [comment]
@modelm in `/frontend-graphql/src` you have `static` symlinked (`static -> ../../frontend/static`) which doesn't work with the frontend-wpgraphql docker container. Maybe just remove the symlink and copy the files in old fashioned.

File: frontend-graphql/src/components/Category.js (hunk 2 above)
Comment (modelm) [reply frontend-graphql/src/components/Category.js]: fixed in https://github.com/postlight/headless-wp-starter/pull/244/commits/7b74cc45ff50be9241622f53680ca30009d42c09

File: frontend-graphql/src/components/Footer.js (hunk 1 above)
Comment (modelm) [reply frontend-graphql/src/components/Footer.js]: fixed by https://github.com/postlight/headless-wp-starter/pull/244/commits/e13d6fd528739e6268458318562adfaeed379339

**Pull request merged**